### PR TITLE
Change gif, video to .webp format to reduce network payload for meshery page

### PR DESCRIPTION
**Pull request by ayushthe1**
**Description**

This PR replaces the gif & video to `.webp ` format as they consume less network payload for meshery page.

This is the network-payload currently :
image

This will be the network payload after merging this PR : 
image

**Notes for Reviewers**
The changes must be verified on safari browser before merging this pr

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d593b22fd294235828ba887a6e8b0156ee58ba65 at: https://64183a3ad057c92d70d8addb--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 0e380c2f00cce71fee3ad98bc05d5eb6d486d7c3 at: https://641841ce65f5501fe720438b--layer5.netlify.app

File: src/sections/Meshery/Features-section/features-section.style.js [hunk 1]
```diff
@@ -47,7 +51,7 @@ const FeaturesSectionWrapper = styled.section`
 		transform: skew(0deg, 6deg);
 		padding: 0 1rem 3rem;
 		text-align: center;
-		h1, h2, p {
+		h1, h2, h3, p {
```
Comment (hirentimbadiya) [inline src/sections/Meshery/Features-section/features-section.style.js:54]: You have added h3 here but haven't used in index.js file

File: src/sections/Meshery/Features-section/features-section.style.js [hunk 2]
```diff
@@ -82,7 +86,7 @@ const FeaturesSectionWrapper = styled.section`
 			margin: 4rem 0;
 			align-items: center;
 		}
-		h2, h4 {
+		h2, h4, h3 {
```
Comment (hirentimbadiya) [inline src/sections/Meshery/Features-section/features-section.style.js:89]: same case here in smp-section added h3 class to add styles but haven't used h3 in index.js file

**Review by hirentimbadiya — COMMENTED** [review]
Overall you converted videos to .webp format from .mp4 but in styling you added header tags but are not used in index.js file , if you can then please make me understand this , otherwise looks good to me.

File: src/sections/Meshery/Features-section/features-section.style.js [hunk 3]
```diff
@@ -242,7 +246,7 @@ const FeaturesSectionWrapper = styled.section`
 	}
 
 	.mesh-mngmnt {
-		h3 {
+		h4 {
```
[the remark this answers is not in the document]
Comment (leecalcote) [reply src/sections/Meshery/Features-section/features-section.style.js:249]: Thanks for pointing this out, @hirentimbadiya 👍

File: src/sections/Meshery/Features-section/index.js [hunk 4]
```diff
@@ -4,7 +4,8 @@ import Button from "../../../reusecore/Button";
 import Slider from "react-slick";
 import "slick-carousel/slick/slick.css";
 import "slick-carousel/slick/slick-theme.css";
-import Slide1 from "../images/service mesh performance example.gif";
+import MeshVideoWebp from "../images/smp-example.webm";
```
Comment (Udit-takkar) [inline src/sections/Meshery/Features-section/index.js:7]: NIT: change the name to MeshVideoWebm

**Review by Udit-takkar — COMMENTED** [review]
@ayushthe1  can we also use StaticImage here https://github.com/layer5io/layer5/blob/0e380c2f00cce71fee3ad98bc05d5eb6d486d7c3/src/sections/Meshery/Features-section/index.js#L25 ?

**Comment by Chadha93** [comment]
Merge conflicts... @ayushthe1

**Comment by Chadha93** [comment]
@ayushthe1 I'm adding this to the agenda, let's discuss this.

**Pull request closed without merging**